### PR TITLE
Updated operating Ceph services to not use DeepSea anymore (gh#366)

### LINKED ISSUE #366: Update ceph unit operations
the format is ceph-$FSID@$DAEMON.$ID
where $FSID is the FSID for the cluster, $DAEMON is the individual daemon (osd, mgr, mon, whatever), and $ID is the ID of that specific daemon
the $ID part can be a bit ugly - it's simple for OSDs (it's just a number) but for other daemons it can have some extra unique identifying junk in there
e.g.: I currently have an MDS, and the service is ceph-af12ebbf-69e9-4286-b8a7-52c0f064bf5c@mds.cephfs.data4.qbmlyg.service

### PR DESCRIPTION
**Pull request by tbazant**
Closes #366

### CODE REVIEW

File: xml/admin_operating_services.xml [hunk 1]
```diff
@@ -10,245 +10,204 @@
...
+    <emphasis>belong to a cluster</emphasis> identified by its
+    <replaceable>FSID</replaceable>.
    </para>
-<screen>&prompt.root;systemctl status ceph-osd@1.service
-&prompt.root;systemctl status ceph-mon@<replaceable>HOSTNAME</replaceable>.service</screen>
    <para>
-    Replace <replaceable>HOSTNAME</replaceable> with the host name the daemon
-    is running on.
+    For example, to stop all &ceph; services on a node regardless to which
+    cluster they belong to, run:
```
Comment (lproven) [inline xml/admin_operating_services.xml:181]: Mismatched numbers: the sentece starts talking about a singular then switches to talking about a plural.

Either:

"all ceph services on nodes regardless of which clusters they belong to"

or:

"all ceph services on a node, regardless of which cluster it belongs to"

File: xml/admin_operating_services.xml [hunk 2]
```diff
@@ -10,245 +10,204 @@
...
-      To keep the downtime low and to find potential issues as early as
-      possible, nodes are restarted sequentially. For example, only one
-      monitoring node is restarted at a time.
-     </para>
-    </listitem>
-   </itemizedlist>
-   <para>
-    The command waits for the cluster to recover if the cluster is in a
-    degraded, unhealthy state.
+    For example, to restart a single OSD service with id 1 on a cluster with ID
```
Comment (lproven) [inline xml/admin_operating_services.xml:199]:
```suggestion
    For example, to restart a single OSD service with ID 1 on a cluster with ID
```

File: xml/bp_troubleshooting_tips.xml [hunk 3]
```diff
@@ -1196,28 +1194,30 @@ ID WEIGHT TYPE NAME    UP/DOWN REWEIGHT PRIMARY-AFFINITY
...
   <para>
-   By default, &deepsea; uses the &adm; as the time server for other
-   cluster nodes. Therefore, if the &adm; is not virtualized, select one or
-   more time servers or pools, and synchronize the local time against them.
-   Verify that the time synchronization service is enabled on each system
-   start-up. Find more information on setting up time synchronization in
+   By default, &deepsea; uses the &adm; as the time server for other cluster
+   nodes. Therefore, if the &adm; is not virtualized, select one or more time
+   servers or pools, and synchronize the local time against them. Verify that
+   the time synchronization service is enabled on each system start-up. Find
```
Comment (lproven) [inline xml/bp_troubleshooting_tips.xml:1200]:
```suggestion
   the time synchronization service is enabled at start-up on each system. Find
```

Comment (lproven) [reply xml/bp_troubleshooting_tips.xml:1200]: Again, just a suggestion.

File: xml/bp_troubleshooting_tips.xml [hunk 4]
```diff
@@ -1429,9 +1431,10 @@ keep_jobs: 1</screen>
   </tip>
  </sect1>
  <sect1 xml:id="large-omap-issues">
-   <title>Large omap Issues</title>
-   <para>
-     Abhi to provide content
-   </para>
+  <title>Large omap Issues</title>
```
Comment (lproven) [inline xml/bp_troubleshooting_tips.xml:1434]:
```suggestion
  <title>Large OMAP Issues</title>
```

File: xml/admin_operating_services.xml [hunk 5]
```diff
@@ -10,245 +10,204 @@
...
-ceph-radosgw.target
-ceph-rbd-mirror.target</screen>
-   <para>
-    To start/stop/restart all &ceph; services on the node, run:
-   </para>
 <screen>
-&prompt.root;systemctl start ceph.target
-&prompt.root;systemctl stop ceph.target
-&prompt.root;systemctl restart ceph.target
+&prompt.cephuser;ceph orch ps
```
Comment (jschmid1) [inline xml/admin_operating_services.xml:29]: Perhaps add a small note that this (and most other) commands support various different formats `[--format json|json-pretty|yaml]`

File: xml/admin_operating_services.xml [hunk 6]
```diff
@@ -10,245 +10,204 @@
...
+   By using the <command>systemctl</command> command, you can operate &ceph;
+   related &systemd; services and targets on a single node.
+  </para>
+
+  <sect2 xml:id="ceph-operating-services-finding-names">
+   <title>Identifying Services and Targets</title>
    <para>
-    You can operate individual services using the following parameterized
-    &systemd; unit files:
+    Before operating &ceph; related &systemd; services and targets, you need to
```
Comment (jschmid1) [inline xml/admin_operating_services.xml:102]: The `NAME` column of `orch ps` is actually the daemon name that can be used with `systemctl`

File: xml/admin_operating_services.xml [hunk 7]
```diff
@@ -10,245 +10,204 @@
...
     To list only the inactive services, run:
    </para>
 <screen>
 &prompt.root;systemctl list-units --all --state=inactive --type=service ceph*
 </screen>
-   <para>
-    You can also use <command>salt</command> to query services across multiple
-    nodes:
-   </para>
+   <tip>
```
Comment (jschmid1) [inline xml/admin_operating_services.xml:158]: It feels that this belongs more in the "Troubleshooting" section. We have a more straightforward way to retrieve all daemons (`ceph orch ps`). We might mislead people by stating this here

File: xml/bp_troubleshooting_tips.xml [hunk 8]
```diff
@@ -1228,21 +1228,22 @@ ID WEIGHT TYPE NAME    UP/DOWN REWEIGHT PRIMARY-AFFINITY
...
     </para>
-    <screen>&prompt.smaster;salt '*' pillar.items</screen>
+<screen>&prompt.smaster;salt '*' pillar.items</screen>
    </listitem>
    <listitem>
     <para>
      Apply the new setting:
     </para>
-    <screen>&prompt.smaster;salt '*' state.apply ceph.time</screen>
+<screen>&prompt.smaster;salt '*' state.apply ceph.time</screen>
```
Comment (jschmid1) [inline xml/bp_troubleshooting_tips.xml:1243]: That's not correct anymore. @ricardoasmarques Is there any 'troubleshooty' way to add timeservers via salt that is worth adding here?

File: xml/admin_operating_services.xml (hunk 5 above)
Comment (tbazant) [reply xml/admin_operating_services.xml:29]: thanks for heads-up

File: xml/admin_operating_services.xml (hunk 6 above)
Comment (tbazant) [reply xml/admin_operating_services.xml:102]: i've tried it now and while `ceph orch ps doc-ses-min2` returns `mon.doc-ses-min2`, `systemctl restart mon.doc-ses-min2` returns an error

Comment (jschmid1) [reply xml/admin_operating_services.xml:102]: what I mean is 

`systemctl status ceph-$fsid@$NAME`

Comment (tbazant) [reply xml/admin_operating_services.xml:102]: great, thanks

File: xml/admin_operating_services.xml (hunk 7 above)
Comment (tbazant) [reply xml/admin_operating_services.xml:158]: ok, commented out for now

File: xml/admin_operating_services.xml (hunk 1 above)
Comment (tbazant) [reply xml/admin_operating_services.xml:181]: best would be "all ceph services on a node regardless of which clusters the services belong to"

File: xml/admin_operating_services.xml [hunk 9]
```diff
@@ -10,245 +10,204 @@
...
+    cluster they belong to, run:
    </para>
+<screen>&prompt.sminion;systemctl stop ceph.target</screen>
    <para>
-    If you do not know the exact name/number of the service, see
-    <xref linkend="ceph-operating-services-finding-names"/>.
+    To restart all services that belong to a &ceph; cluster with ID
+    <literal>b4b30c6e-9681-11ea-ac39-525400d7702d</literal>, run:
    </para>
+<screen>&prompt.sminion;systemctl restart ceph-b4b30c6e-9681-11ea-ac39-525400d7702d.target</screen>
```
[the remark this answers is not in the document]
Comment (tbazant) [reply xml/admin_operating_services.xml:208]: @jschmid1 do we have other 'high-level' command that can restart all ceph daemons on a node?

[the remark this answers is not in the document]
Comment (jschmid1) [reply xml/admin_operating_services.xml:208]: Not on a per host basis, just on a per service basis.

**Comment by asettle** [comment]
> Pending "TBD" and "Abhi to provide" §s, OK.

They'll be from the troubleshooting guide and that's fine :)

**Pull request merged**